scripts/release_storybook_static

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+
+set -eu -o pipefail
```

Comment:
```suggestion
set -u
```

If line 22 does return an error status, the output will be consumed by the assignment expression, the script will terminate before we reach line 28, and so the output (with the error message) will never be printed. Since we have explicit error handling `-e` does not help, but actually terminates the script before the error handling can run.